packages/api/src/http/routes/user.ts

```diff
@@ -19,6 +19,11 @@ const schema_register = {
 export default function(di: DIContainer, oauth: OAuthServer): Router {
 	const router = Router()
 	const controller = genController(di)
+	
+	router.get('/auth', oauth.authorize({
```

Comment:
Why is this GET ?

Reply:
Because of NuxtJS. The auth module redirects when using the Oauth2 scheme.